projects/gateway2/translator/plugins/listeneroptions/listener_options_plugin_test.go

```diff
@@ -114,11 +122,10 @@ var _ = Describe("ListenerOptions Plugin", func() {
 			It("does not add buffer limit", func() {
 				err := plugin.ApplyListenerPlugin(ctx, listenerCtx, outputListener)
 				Expect(err).ToNot(HaveOccurred())
-				Expect(outputListener.GetOptions()).To(BeNil())
+				Expect(outputListener.GetOptions().GetPerConnectionBufferLimitBytes()).To(BeNil())
```

Comment:
can you add a test that shows the merging, (if there isn't already one)?

Reply:
it's not super explicit; We now have ProxyProtocol set on the base options, expect `outputListener.GetOptions() == expectedOptions` encodes this. 

I can rewrite the tests to be super explicit if we want that.